ports/mimxrt/hal/qspi_nor_flash_config.c

```diff
@@ -129,11 +129,17 @@ const flexspi_nor_config_t qspiflash_config = {
             FLEXSPI_LUT_SEQ(0, 0, 0, 0, 0, 0),         // Filler
             FLEXSPI_LUT_SEQ(0, 0, 0, 0, 0, 0),         // Filler
             FLEXSPI_LUT_SEQ(0, 0, 0, 0, 0, 0),         // Filler
+
+            // 13 Erase Block (32k) -> 13
+            FLEXSPI_LUT_SEQ(CMD_SDR, FLEXSPI_1PAD, 0x52, RADDR_SDR, FLEXSPI_1PAD, 24),
+            FLEXSPI_LUT_SEQ(0, 0, 0, 0, 0, 0),         // Filler
+            FLEXSPI_LUT_SEQ(0, 0, 0, 0, 0, 0),         // Filler
+            FLEXSPI_LUT_SEQ(0, 0, 0, 0, 0, 0),         // Filler
         },
     },
     .pageSize = 256u,
     .sectorSize = 4u * 1024u,
-    .blockSize = 64u * 1024u,
+    .blockSize = 32u * 1024u,
```

Comment:
This looks like a bug fix?

Reply:
This change accompanies the addition of the actual `flash_erase_block` function with the matching look up table entry `13` which you can find above in line 132 ff.

I decided to implement the block erase command `0x52` which erases a 32Kbyte block of memory. Hence the modification/correction.

As explained in another answer, both 32 and 64 Kbyte are basically true since the flash chips support two block erase operations (`0x52` = 32KByte and `0xD8` = 64Kbyte). However I decided to configure the block size which can actually be erased by the respective operation.

Reply:
OK!